public/js/buffalo/buffalo.js

```diff
@@ -0,0 +1,71 @@
+let testCount = 0
+
+function describe(testGroup, it) {
+  addGroupToHtml(testGroup);
+  try {
+    return it();
+  } catch(err) {
+    addErrorToGroup(err.message);
+  }
+}
+
+updateResultCount();
+
+function it(test, expectation) {
```

Comment:
the argument names surprised me a little -- I think someting like `function it(test_description, test)` sounds more natural to me as `expectation` makes me think of only the line that says `expect(...)`